### ISSUE TITLE
ImportError: No module named 'sentinel'

### ISSUE THREAD

**Issue by zhangsm** [celery/celery#3996]
```python
redis_url = 'sentinel://%s:%s;' % (REDIS_SENTINEL_HOST, REDIS_SENTINEL_PORT)
celery = Celery('server', broker=redis_url, backend=redis_url)
```

```bash
[2017-04-25 11:26:11,596: CRITICAL/MainProcess] Unrecoverable error: ImportError("No module named 'sentinel'",)
Traceback (most recent call last):
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/kombu/utils/objects.py", line 42, in __get__
    return obj.__dict__[self.__name__]
KeyError: 'backend'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/worker/worker.py", line 203, in start
    self.blueprint.start(self)
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/bootsteps.py", line 115, in start
    self.on_start()
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/apps/worker.py", line 143, in on_start
    self.emit_banner()
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/apps/worker.py", line 158, in emit_banner
    ' \n', self.startup_info(artlines=not use_image))),
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/apps/worker.py", line 221, in startup_info
    results=self.app.backend.as_uri(),
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/kombu/utils/objects.py", line 44, in __get__
    value = obj.__dict__[self.__name__] = self.__get(obj)
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/app/base.py", line 1182, in backend
    return self._get_backend()
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/app/base.py", line 900, in _get_backend
    self.loader)
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/app/backends.py", line 65, in by_url
    return by_name(backend, loader), url
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/celery/app/backends.py", line 45, in by_name
    cls = symbol_by_name(backend, aliases)
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/usr/local/var/pyenv/versions/3.5.2/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'sentinel'
```

how can i solve?

**Comment by geoffreybauduin** [comment]
See #4053

**Comment by thedrow** [comment]
Fixed by #4144.
